uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -223,7 +223,7 @@ Feature: GGMQ-1
 }
     """
     And I deploy the Greengrass deployment configuration
-    Then the Greengrass deployment is COMPLETED on the device after 2 minutes
+    Then the Greengrass deployment is COMPLETED on the device after 299 seconds
```

Comment:
why?

Reply:
At first deployment we use 5 minutes timeout equals to 300 seconds.  299 used to easy distinguishing first from second with almost the same timeout. When run scenarios locally sometimes 2 minutes is not enough.